lib/enkf/tests/enkf_gen_data_config_parse.cpp

```diff
@@ -152,7 +149,7 @@ enkf_config_node_type * parse_alloc_GEN_DATA_result( const char * config_string
 
 
 void test_parse_gen_data_result() {
-  test_work_area_type * work_area = test_work_area_alloc("GEN_DATA_RESULT_parse");
+  ecl::util::TestArea ta("GEN:_DATA_RESULT_parse");
```

Comment:
there is a colon in the name, I think some OSs are not too happy with having a colon in a folder name (and if the tests succeeded, maybe it's worth investigating this further)

Reply:
Colon removed in fixup